### DISCUSSION TITLE
Just download the relevant part!

### DISCUSSION THREAD

**Discussion by emredarak** [locus84/Locus-Bundle-System#16] [Ideas]
I am modifying an object in the assets bundle and it downloads all the assets bundle content i.e. it downloads the unchanged ones as well. Only the relevant part is not available or can't be downloaded?

**Comment by locus84** [comment]
It already provides that feature, but it also download if a bundle has dependency on the changed bundle.

**Reply by emredarak** [reply]
"BundleInfos": [
        {
            "BundleName": "Remote",
            "m_HashString": "1cde14e5341c0da35bca7f28e780dec5",
            "Dependencies": [],
            "Size": 110644
        }

There are 3 cubes inside

I only changed 1 cube but the downloaded size.

Need to download 0,110644 mb
UnityEngine.Debug:Log (object)
TitleScene/<Start>d__0:MoveNext () (at Assets/Scripts/TitleScene.cs:21)
UnityEngine.SetupCoroutine:InvokeMoveNext (System.Collections.IEnumerator,intptr)

Need to download 0,110644 mb

**Reply by locus84** [reply]
You should define multiple bundles for that. A bundle means a file and if an included asset has changed, then it also changes the bundle.
